contracts/zero-ex/foundry.toml

```diff
@@ -3,5 +3,24 @@ src = 'contracts/src'
 out = 'foundry-artifacts'
 test = 'tests'
 libs = ["contracts/deps/", "../utils/contracts/src/"]
-remappings = ['@0x/contracts-utils/=../utils/', '@0x/contracts-erc20/=../erc20/', 'src/=./contracts/src']
+remappings = [
+    'forge-std/=contracts/deps/forge-std/src/',
+    'ds-test/=contracts/deps/forge-std/lib/ds-test/src/',
+    '@0x/contracts-utils/=../utils/',
+    '@0x/contracts-erc20/=../erc20/',
+    'src/=./contracts/src',
+    'samplers/=contracts/test/samplers',
+    'utils/=./tests/utils',
+]
 optimizer_runs = 1000000
+fs_permissions = [{ access = "read", path = "./" }]
```

Comment:
Is there a way to selectively configure whether to use the IR optimizer if the solc version is high enough?